e2e/categories.spec.ts

```diff
@@ -23,8 +26,20 @@ electronTest('categories', async (app) => {
       dialog.locator('span', { hasText: 'Great games' })
     ).toBeInViewport()
 
-    // rename category
+    // add button should be disable if trying to rename category to empty name
```

Comment:
Here you are testing that the button is disabled when the name is unchanged, the comment doesn't match that

the other 2 expects are for the blank tests

I can merge this after that's fixed

Reply:
Sorry for the delay in fixing this.
These previous days were very busy.
I fixed this now. Where it says "empty" should be "same", I didn't realize that "words swap".